cypress/e2e/shared/flows.test.ts

```diff
@@ -36,63 +36,40 @@ describe('Flows', () => {
       .click()
     cy.getByTestID('renamable-page-title--input').type('My Flow {enter}')
 
+    // "Add Another Panel" menu is present and there is a Query Builder button
+    cy.get('.insert-cell-menu.always-on').contains('Add Another Panel')
+    cy.getByTestID('add-flow-btn--queryBuilder').should('have.length', 1)
+
     cy.getByTestID('sidebar-button')
       .first()
       .click()
     cy.getByTestID('Delete--list-item').click()
 
-    cy.getByTestID('panel-add-btn--1').click()
-    cy.getByTestID('add-flow-btn--queryBuilder').click()
-    cy.getByTestID('selector-list defbuck')
-      .first()
-      .click()
-    cy.getByTestID('selector-list test')
+    cy.get('.flow-divider--button')
       .first()
       .click()
 
-    cy.getByTestID('time-machine-submit-button').click()
-
-    cy.getByTestID('panel-add-btn-1').click()
+    // Opening the menu adds another Query Builder button
+    cy.getByTestID('add-flow-btn--queryBuilder').should('have.length', 2)
 
-    cy.getByTestID('add-flow-btn--visualization').click()
-  })
-
-  // NOTE: we are hiding the metric selector from users for now
-  it.skip('can create a bucket from the metric selector and verify it is selected', () => {
```

Comment:
The diff looks strange here but the entire skipped test is deleted. Flows tests use a new pattern to avoid flakiness. There is no point in updating a skipped test. When desired in the future, add the test back following the pattern found in the other tests.